### ISSUE TITLE
Replace (proxy) at all body links 

### ISSUE THREAD

**Issue by faustoct** [waterlink/rack-reverse-proxy#22]
first of all thanks for your gem. 

when i use the gem at mydomain.com/blog if proxies perfect pointing to my mydomain.wordpress.com blog. The problem is when I click any link it redirect to my mydomain.wordpress.com/link . I'd like to keep it under same domain for every click.

**Comment by waterlink** [comment]
The library works correctly. It does not do anything to the payload, that is transferred. There are 2 possible solutions:

- Try to make all urls relative instead of absolute on the original website, i.e.: instead of `http://example.com/the/url/path/here` it should just have `/the/url/path/here`. Or
- Add additional `Rack` middleware, that would make a necessary rewrite of all links on the html pages.

**Comment by faustoct** [comment]
great. thanks my friend.

**Comment by waterlink** [comment]
Yes, either change the source to use relative links, or add a Rack middleware, that will rewrite your HTML.

**Comment by kaspergrubbe** [comment]
Could someone maybe present som example code to do this? I think this is very a very common use-case.
